automated install/basic-install.sh

```diff
@@ -1580,6 +1580,31 @@ accountForRefactor() {
 }
 
 updatePihole() {
+  # Create the pihole user
+  create_pihole_user
+
+  # If the user wants to install the Web interface,
+  if [[ "${INSTALL_WEB}" == true ]]; then
+    if [[ ! -d "/var/www/html" ]]; then
+      # make the Web directory if necessary
+      mkdir -p /var/www/html
+    fi
+    # Set the owner and permissions
+    chown ${LIGHTTPD_USER}:${LIGHTTPD_GROUP} /var/www/html
+    chmod 775 /var/www/html
+    # Give pihole access to the Web server group
+    usermod -a -G ${LIGHTTPD_USER} pihole
+    # If the lighttpd command is executable,
+    if [[ -x "$(command -v lighty-enable-mod)" ]]; then
+      # enable fastcgi and fastcgi-php
+      lighty-enable-mod fastcgi fastcgi-php > /dev/null || true
+    else
+      # Otherwise, show info about installing them
+      echo -e  "  ${INFO} Warning: 'lighty-enable-mod' utility not found
+      Please ensure fastcgi is enabled if you experience issues\\n"
+    fi
+  fi
```

Comment:
This section of code was copy-pasted, so can you make it into a function and call it instead of copying it in two locations?

Reply:
Sure thing.